packages/error-reporting/src/interfaces/manual.js

```diff
@@ -74,6 +74,10 @@ function handlerSetup(client, config) {
     }
 
     if (err instanceof ErrorMessage) {
+      var fauxError = new Error(err.message);
+      var fullStack = fauxError.stack.split('\n');
+      var cleanedStack = fullStack.slice(0, 1).concat(fullStack.slice(2)).join('\n');
```

Comment:
I think `fullStack.splice(1, 1)` may make it more clear what this does. Could you add a comment describing why only the second frame needs to be removed?